.github/workflows/tests.yml

```diff
@@ -58,12 +58,17 @@ jobs:
       with:
         submodules: true
 
-    - uses: pypa/cibuildwheel@v2.8.1
+    - uses: pypa/cibuildwheel@v2.9.0
       env:
         CIBW_ARCHS_MACOS: universal2
         CIBW_MANYLINUX_X86_64_IMAGE: manylinux1
         CIBW_BUILD: cp39-win_amd64 cp36-manylinux_x86_64 cp38-macosx_universal2
 
+    - uses: pypa/cibuildwheel@v2.9.0
+      if: matrix.os == 'ubuntu-latest'
+      env:
+        CIBW_BUILD: cp311-manylinux_x86_64
```

Comment:
We are hardcoding manylinux1 above, and manylinux1 doesn't support 3.11 (or 3.10, IIRC), so made this a separate step.

Reply:
Is this what becomes easier when we drop Python 3.6? I'm going to open issues for dropping 3.6 in Uproot and Awkward as a reminder.